src/main/webapp/app/overview/course-competencies/course-competencies.component.ts

```diff
@@ -43,6 +44,7 @@ export class CourseCompetenciesComponent implements OnInit, OnDestroy {
         private courseStorageService: CourseStorageService,
         private competencyService: CompetencyService,
         private prerequisiteService: PrerequisiteService,
+        private courseCompetencyService: CourseCompetencyService,
```

Comment:
do we need all three services, or is the CourseCompetencyService not enough for this use case?